docs/development/docs.rst

```diff
@@ -10,27 +10,50 @@ The docs are kept in the ``docs/`` directory at the top of the source tree.
    However, if there is going to be periodic/bigger contributions, it may be a
    good idea to install the Sphinx requirements to build our docs.
 
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation, you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
```

Comment:
For long lines, you can break them after the comma https://docs.readthedocs.io/en/stable/development/docs.html#content